src/compiler/typescript_to_rust/azle_generate/src/ts_ast/ts_type/to_candid_type.rs

```diff
@@ -21,6 +21,15 @@ impl SourceMapped<'_, TsType> {
             TsType::TsTupleType(_) => {
                 return Err(vec![UnexpectedTsTupleTypes::from_ts_type(self).into()]);
             }
+            TsType::TsArrayType(_) => {
+                // TODO: Because this isn't handled types like this currently break:
```

Comment:
I don't think this needs to be fixed right? You have to use `Vec`

Reply:
Correct. I've deleted this.